src/components/list/list.scss

```diff
@@ -0,0 +1,125 @@
+@import "variables";
+
+$md-list-side-padding: 16px;
+$md-list-avatar-size: 40px;
+
+/*  Normal list variables */
+$md-list-top-padding: 8px;
+$md-list-font-size: 16px;
+$md-list-secondary-font: 14px;
+$md-list-base-height: 48px;
+$md-list-avatar-height: 56px;
+$md-list-two-line-height: 72px;
+$md-list-three-line-height: 88px;
+
+/* Dense list variables */
+$md-dense-top-padding: 4px;
+$md-dense-font-size: 13px;
+$md-dense-base-height: 40px;
+$md-dense-avatar-height: 48px;
+$md-dense-two-line-height: 60px;
+$md-dense-three-line-height: 76px;
+
+@mixin md-list-item-base($font-size, $base-height, $avatar-height, 
```

Comment:
It might be good for these mixins to describe what's captured in the `base` and what is left to the specific uses.